sub/source/docs/modes.rst

```diff
@@ -0,0 +1,62 @@
+.. _modes:
+
+Sub Modes
+=========
+Sub has the following modes:
+
+legend for requirements: P - needs Position(GPS,etc.), D - needs Depth sensor
+
+=============   =================================================  =========
+Mode            Description                                        Requires
+=============   =================================================  =========
+MANUAL          Pilot control with no stabilization                 -
+ACRO            Body-frame rate control, manual depth control       -
+STABILIZE       Manual angle control,manual depth control           -
+ALT_HOLD        Stabilize with automatic depth control              D
+AUTO            Automatic mission command list execution            P/D
+GUIDED          Swim to location or velocity/direction using GCS    P/D
+CIRCLE          Circle swim with depth control                      P/D
+SURFACE         Return to surface, pilot directional control        P/D
+POSHOLD         Loiter with depth control and pilot overrides       P/D
+MOTOR_DETECT    Automatically determine motor rotation and adjust   D
+SURFTRAK        Hold distance above seafloor while stabilizing      -
```

Comment:
```suggestion
SURFTRAK        Hold distance above seafloor while stabilizing      R
```
We likely need an additional "requires" designation here, because it expects a range-finder for maintaining relative altitude/distance :-)

Reply:
done